lxd/device/unix_hotplug.go

```diff
@@ -149,29 +148,36 @@ func (d *unixHotplug) Start() (*deviceConfig.RunConfig, error) {
 	runConf := deviceConfig.RunConfig{}
 	runConf.PostHooks = []func() error{d.Register}
 
-	device := d.loadUnixDevice()
-	if d.isRequired() && device == nil {
+	devices := d.loadUnixDevices()
+	if d.isRequired() && devices == nil {
 		return nil, fmt.Errorf("Required Unix Hotplug device not found")
 	}
 
-	if device == nil {
+	if devices == nil {
```

Comment:
do we need this anymore? If the devices var is nil or zero length then the for loop wont iterate it and the return statement at the bottom will run as usual.

Reply:
You're right, we don't need this check anymore. Good catch.